std_msgs/Header header

geometry_msgs/Vector3 pos
float64 wrist_angle
float64 wrist_roll
float64 gripper

bool is_at_target